mote/__init__.py

```diff
@@ -29,7 +29,7 @@
 
 import flask, random, string, json, util, re
 import dateutil.parser, requests, collections
-import logging
+import logging, soke
```

Comment:
Because it seems to be only used in Except clauses, so if you never faced the exception, it was never called :)